sdk/Trace/ExporterFactory.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Trace;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\HttpFactory;
+use Nyholm\Dsn\DsnParser;
+use OpenTelemetry\Contrib\Jaeger\Exporter as JaegerExporter;
+use OpenTelemetry\Contrib\Newrelic\Exporter as NewrelicExporter;
+use OpenTelemetry\Contrib\Otlp\Exporter as OtlpExporter;
+use OpenTelemetry\Contrib\OtlpGrpc\Exporter as OtlpGrpcExporter;
+use OpenTelemetry\Contrib\Zipkin\Exporter as ZipkinExporter;
+use OpenTelemetry\Contrib\ZipkinToNewrelic\Exporter as ZipkinToNewrelicExporter;
+
+class ExporterFactory
+{
+    private $name;
```

Comment:
If the upstream repo has different standards, ignore this, but I've always prefixed protected and private member variables/methods with `_` to ensure it is clear that an internal member is being used.

```suggestion
    private $_name;
```